packages/keyv/src/index.d.ts

```diff
@@ -16,6 +16,8 @@ declare namespace Keyv {
 		deserialize?: ((data: string) => any | undefined) | undefined;
 		ttl?: number | undefined;
 	}
+
+	type Store<T> = Keyv<T>;
```

Comment:
Is this accurate? In the existing types `Store` was a simpler interface. Does it actually need to be a `Keyv`?